start/src/main/java/org/apache/accumulo/start/classloader/vfs/AccumuloReloadingVFSClassLoader.java

```diff
@@ -199,6 +200,9 @@ public AccumuloReloadingVFSClassLoader(String uris, FileSystemManager vfs,
    * thread
    */
   public void close() {
+    for (FileObject file : Arrays.asList(this.files)) {
+      monitor.removeFile(file);
```

Comment:
Interesting fix. Can you explain how this prevents the RuntimeException?  Is the upgrade to 2.3 needed for this change?

Reply:
I was curious about that, too. I'm not even sure we ever bother to close this ClassLoader, so I'm not even sure this would run.